reviewing/dependencies-hierarchy/src/index.ts

```diff
@@ -236,26 +224,47 @@ interface GetTreeOpts {
   wantedPackages: PackageSnapshots
 }
 
-interface DependencyInfo { circular?: true, dependencies: PackageNode[] }
+interface DependencyInfo {
+  dependencies: PackageNode[]
+
+  circular?: true
+
+  /**
+   * Whether or not the dependencies array was fully enumerated. This may not be
+   * the case if a max depth was hit.
+   */
+  isPartiallyVisited: boolean
```

Comment:
I decided to make this required to prevent code in `getTreeHelper` from accidentally forgetting to set this field, which I could see happening after future refactors to this function. The lack of `?` here forces each creation of `DependencyInfo` to explicitly declare this field.